frontend/src/metabase/collections/components/CreateCollectionForm/tests/setup.tsx

```diff
@@ -0,0 +1,54 @@
+/* istanbul ignore file */
```

Comment:
Please rename to `setup.spec.tsx` because of https://github.com/metabase/metabase/pull/32164